src/rgw/rgw_sal_motr.cc

```diff
@@ -1550,11 +1550,8 @@ int MotrObject::MotrReadOp::prepare(optional_yield y, const DoutPrefixProvider*
 
   // Set source object's attrs. The attrs is key/value map and is used
   // in send_response_data() to set attributes, including etag.
-  bufferlist etag_bl;
   string& etag = ent.meta.etag;
   ldpp_dout(dpp, 20) <<__func__<< ": object's etag: " << ent.meta.etag << dendl;
-  etag_bl.append(etag.c_str(), etag.size());
-  source->get_attrs().emplace(std::move(RGW_ATTR_ETAG), std::move(etag_bl));
```

Comment:
Was this overwriting all other Obj attributes ?
Please state reasons on how this impacts

Reply:
No this was not overwriting, but with the other change this became redundant. As of Now I have put it back in case someone wants to implement this here. also it will not be an overhead as map.emplace will do nothing if etag's key is present already.

Other solution would be to set all other meta keys just like RGW_ATTR_ETAG here ifself.

Reply:
lets not touch any code unrelated to PR or tasks. Such clean ups should be targeted when we plan for their testing too. as i see you have reverted this code